workos/organizations.py

```diff
@@ -29,77 +34,12 @@ def request_helper(self):
 
     def list_organizations(
         self,
-        domains=None,
-        limit=None,
-        before=None,
-        after=None,
-        order=None,
-    ):
-        """Retrieve a list of organizations that have connections configured within your WorkOS dashboard.
-
-        Kwargs:
-            domains (list): Filter organizations to only return those that are associated with the provided domains. (Optional)
-            limit (int): Maximum number of records to return. (Optional)
-            before (str): Pagination cursor to receive records before a provided Organization ID. (Optional)
-            after (str): Pagination cursor to receive records after a provided Organization ID. (Optional)
-            order (Order): Sort records in either ascending or descending order by created_at timestamp.
-
-        Returns:
-            dict: Organizations response from WorkOS.
-        """
-        warn(
-            "The 'list_organizations' method is deprecated. Please use 'list_organizations_v2' instead.",
-            DeprecationWarning,
-        )
-        if limit is None:
-            limit = RESPONSE_LIMIT
-            default_limit = True
-
-        params = {
-            "domains": domains,
-            "limit": limit,
-            "before": before,
-            "after": after,
-            "order": order or "desc",
-        }
-
-        if order is not None:
-            if isinstance(order, Order):
-                params["order"] = str(order.value)
-
-            elif order == "asc" or order == "desc":
-                params["order"] = order
-            else:
-                raise ValueError("Parameter order must be of enum type Order")
-
-        response = self.request_helper.request(
-            ORGANIZATIONS_PATH,
-            method=REQUEST_METHOD_GET,
-            params=params,
-            token=workos.api_key,
-        )
-
-        response["metadata"] = {
-            "params": params,
-            "method": Organizations.list_organizations,
-        }
-
-        if "default_limit" in locals():
-            if "metadata" in response and "params" in response["metadata"]:
-                response["metadata"]["params"]["default_limit"] = default_limit
-            else:
-                response["metadata"] = {"params": {"default_limit": default_limit}}
-
-        return response
-
-    def list_organizations_v2(
-        self,
-        domains=None,
-        limit=None,
-        before=None,
-        after=None,
-        order=None,
-    ):
+        domains: Optional[List[str]] = None,
+        limit: Optional[int] = None,
```

Comment:
If `limit` has a default value of `RESPONSE_LIMIT` below, any reason not to make that the default value here?
```suggestion
        limit: Optional[int] = RESPONSE_LIMIT,
```

Reply:
... and remove the `default_limit = True` below. It doesn't seem to do anything.